pkg/runtime/runtime.go

```diff
@@ -29,6 +29,8 @@ const (
 	Skipped
 )
 
+type Filters []string
```

Comment:
Small point. Would this be better in `suite.go`?

Reply:
I placed it in `runtime` because the filtering logic should be inside the runtime in the future. 
The suite would be the wrong place because the filters are not defined in the suite, instead filters are only provided viá the command line.